doc/source/ray-contribute/development.rst

```diff
@@ -14,103 +14,58 @@ For a majority of Ray users, installing Ray via the latest wheels or pip package
 .. contents::
   :local:
 
-Clone the repository
+Fork the Ray repository
 --------------------
 
-To build Ray locally you will need to have the Git repository, so first, fork it on GitHub. Then you can clone it to your machine:
+Forking an open source repository is a best practice when looking to contribute, as it allows you to make and test changes without affecting the original project, ensuring a clean and organized collaboration process.
+You can propose changes to the main project by submitting a pull request to the main project's repository.
 
-.. tab-set::
```

Comment:
This removal may be slightly controversial, but my line of thought here is that the GitHub documentation is so clear, and it effectively covers Mac, Windows, Linux, GitHub CLI, Desktop, and Web browser methods. In contrast, the portion I removed only covers one method of cloning instructions and doesn't address forking. Why try to provide an in-place watered down version of really really well-done GitHub docs?